cmd/fabric-ca-server/config.go

```diff
@@ -576,6 +576,30 @@ func (s *ServerCmd) configInit() (err error) {
 		return err
 	}
 
+	// Read operations tls files
+	if s.myViper.GetString("operations.tls.enabled") == "true" {
+		cf := s.myViper.GetString("operations.tls.cert.file")
+		if util.FileExists(cf) {
+			if filepath.IsAbs(cf) {
+				s.cfg.Operations.TLS.CertFile = cf
+			} else {
+				s.cfg.Operations.TLS.CertFile = filepath.Join(filepath.Dir(s.cfgFileName), cf)
```

Comment:
This is making an assumption about the working directory. If the cert file is relative to the config path but not the current working directory, this won't work.

Honestly, I don't know why we need to determine if the file exists before trying to use it. Checking if it's absolute or relative is important, and then, when relative, that we make it relative to the config file location.

A missing file should result in a config read error.